Tests/MapboxNavigationTests/NavigationGeocodedPlacemarkTests.swift

```diff
@@ -0,0 +1,85 @@
+import XCTest
+import CarPlay
+import MapboxNavigation
+@testable import TestHelper
+
+class NavigationGeocodedPlacemarkTests: TestCase {
+    
+    @available(iOS 12.0, *)
+    func testNavigationGeocodedPlacemarkListItem() {
+        
+        let title = "title"
+        let subtitle = "subtitle"
+        let navigationGeocodedPlacemark = NavigationGeocodedPlacemark(title: title,
+                                                                      subtitle: subtitle,
+                                                                      location: nil,
+                                                                      routableLocations: nil)
+        
+        let listItem = navigationGeocodedPlacemark.listItem()
+        XCTAssertEqual(listItem.text, title, "Titles should be equal.")
+        XCTAssertEqual(listItem.detailText, subtitle, "Subtitles should be equal.")
+        XCTAssertEqual(listItem.image, nil, "Image should not be set.")
+    }
+    
+    func testNavigationGeocodedPlacemarkEncodingAndDecoding() {
+        let coordinate = CLLocationCoordinate2D(latitude: 0.0, longitude: 0.0)
```

Comment:
I think makes sense to change lat/lon to non zero values. 0 is a default value for primitive double values.

Reply:
Don't think it's too critical at this point. Will leave it as it is.